src/screens/FeedScreen/RenderList.js

```diff
@@ -146,6 +149,7 @@ const RenderListFeed = (props) => {
             messageContainerStyle={{paddingHorizontal: 10}}
             topics={item?.topics}
             contentHeight={contentLinkHeight()}
+            hasComment={hasComment}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `hasComment` prop is being passed to the `ContentLink` component. Ensure that this prop is used correctly in the `ContentLink` component and that it doesn't conflict with any other props with the same name.
```diff
+ hasComment={hasComment}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->